src/Core/Settings.cpp

```diff
@@ -5970,6 +5970,10 @@ This only affects operations performed on the client side, in particular parsing
 Normally this setting should be set in user profile (users.xml or queries like `ALTER USER`), not through the client (client command line arguments, `SET` query, or `SETTINGS` section of `SELECT` query). Through the client it can be changed to false, but can't be changed to true (because the server won't send the settings if user profile has `apply_settings_from_server = false`).
 
 Note that initially (24.12) there was a server setting (`send_settings_to_client`), but latter it got replaced with this client setting, for better usability.
+)", 0) \
+    \
+    DECLARE(Bool, query_plan_join_swap_table_use_statistics, true, R"(
```

Comment:
Tbh, not sure that whether it should be enabled by default

Reply:
Perhaps we could use a more generic name, such as `query_plan_join_use_statistics`, since we may use statistics for other join-related optimizations in the future. Introducing a separate setting like `query_plan_join_reordering_use_statistics/query_plan_join_algorithm_use_statistics/...` seems overly granular, imo